src/Components/Publishing/Constants.ts

```diff
@@ -162,12 +162,14 @@ export const getArtsySlugsFromArticle = (
   article: ArticleData
 ): SlugsFromArticle => {
   const articleBody = article.sections
-    .map(section => {
-      if (section.type === "text") {
-        return section.body
-      }
-    })
-    .join()
+    ? article.sections
+        .map(section => {
+          if (section.type === "text") {
+            return section.body
+          }
+        })
+        .join()
+    : ""
```

Comment:
In the old setup, we never wrapped `series` layouts, which have no `sections`. This is a minor change to make sure the provider doesn't fail on articles without `sections`.